app/components/pipeline/Header/index.js

```diff
@@ -189,7 +174,7 @@ class Header extends React.Component {
     if (this.props.pipeline.description) {
       return <Emojify text={this.props.pipeline.description} />;
     } else {
-      return this.repositoryUrl();
+      return this.props.pipeline.repository.url;
```

Comment:
Yeah, but this is just displayed as text if there's no description…

<img width="338" alt="this is just shown here" src="https://cloud.githubusercontent.com/assets/153/25831579/849df766-34a8-11e7-9d2a-85bf3f66658a.png">